go/libkb/chain_link_v2.go

```diff
@@ -157,7 +157,7 @@ func (t SigchainV2Type) RequiresAtLeastRole() keybase1.TeamRole {
 }
 
 func (t SigchainV2Type) TeamAllowStubWithAdminFlag(isAdmin bool) bool {
-	role := keybase1.TeamRole_READER
+	role := keybase1.TeamRole_BOT
```

Comment:
There's something pre-existingly weird about these methods. I think `TeamAllowStub` is unused. If you'd like you can merge them and I think `role` goes away.